src/contracts/data-bus/data-bus.client.ts

```diff
@@ -29,10 +32,21 @@ export class DataBusClient {
   }
 
   async sendTransaction(eventId: string, dataBytes: string) {
+    const estimatedGas = await this.dataBus.estimateGas.sendMessage(
+      eventId,
+      dataBytes,
+    );
+
+    const gasLimit = estimatedGas.mul(DATA_BUS_GAS_MULTIPLIER).div(100);
```

Comment:
Optional:
in depositor bot we have sanity check `CONTRACT_GAS_LIMIT` to be sure we won't spent more fee than we expect